api/dbadapters/index.ts

```diff
@@ -32,7 +32,7 @@ export function create(profile: protos.IProfile, warehouseType?: string): DbAdap
     return new registry[warehouseType](profile);
   }
   if (!!profile.bigquery) {
-    return new registry["bigquery"](profile);
+    throw new Error(profile.toString());
```

Comment:
This doesn't look right...

Reply:
Oops, sorry, debugging stuff, didn't intend it to make it here.